nilearn/datasets.py

```diff
@@ -439,7 +455,8 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
     try:
         # Download data
         displayed_url = urllib.splitquery(url)[0] if verbose == 0 else url
```

Comment:
I guess this one should become `if verbose == 1` (shortening of the url for easier reading with the default verbosity)